solutions/fully-configurable/DA-types.md

```diff
@@ -0,0 +1,125 @@
+# Configuring complex inputs for Cloud Automation for Cloud Logs
+
+Several optional input variables in the IBM Cloud [Cloud Logs instances deployable architecture](https://cloud.ibm.com/catalog#deployable_architecture) use complex object types. You specify these inputs when you configure deployable architecture.
+
+* Cloud Logs Event Notification Instances (`cloud_logs_existing_en_instances`)
+* Cloud Logs policies (`cloud_logs_policies`)
+* Cloud Logs data bucket retention policy(`cloud_log_data_bucket_retention_policy`)
+
+
+## Cloud Logs Event Notification Instances <a name="cloud_logs_existing_en_instances"></a>
+
+The `cloud_logs_existing_en_instances` input variable allows you to provide a list of existing Event Notification (EN) instances that will be integrated with the Cloud Logging service. For each EN instance, you need to specify its CRN (Cloud Resource Name) and region. You can optionally configure a integration name and control whether to skip the creation of an authentication policy for the instance. You can also optionally configure the notifications routing through email per instance.
+
+- Variable name: `cloud_logs_existing_en_instances`.
+- Type: A list of objects. Each object represents an EN instance.
+- Default value: An empty list (`[]`).
+
+### Options for cloud_logs_existing_en_instances
+
+  - `instance_crn` (required): The Cloud Resource Name (CRN) of the Event Notification instance.
+  - `integration_name` (optional): The name of the Event Notification integration that gets created. If a prefix input variable is passed, it is prefixed to the value in the `<prefix>-value` format. Defaults to `"cloud-logs-en-integration"`.
+  - `skip_en_auth_policy` (optional): A boolean flag to determine whether to skip the creation of an authentication policy that allows Cloud Logs 'Event Source Manager' role access in the existing event notification instance. Defaults to `false`.
+  - `from_email` (optional): The `from` email address used in any Cloud Logs events coming via Event Notifications. Defaults to `cloudlogsalert@ibm.com`.
+  - `reply_to_email` (optional): The `reply_to` email address used in any Cloud Logs events coming via Event Notifications. Defaults to `no-reply@ibm.com`.
+  - `email_list` (optional): The list of email addresses to notify when Cloud Logs triggers an event. Defaults to an empty list.
+
+### Example Event Notification Instance Configuration
+
+```hcl
+cloud_logs_existing_en_instances = [
```

Comment:
The code block should not include `cloud_logs_existing_en_instances =` in it. It should just include the value of the variable so users can click the copy to clipboard in the UI and paste the exact value